Tasks/Xcode/task.json

```diff
@@ -95,7 +90,6 @@
             "required": true,
             "helpMarkDown": "Choose the tool to use for generating the IPA.",
```

Comment:
We should be able to get rid of this combo since there will be only one option left after removing xcrun

Reply:
👍 Whoops!